org.testeditor.web.backend.persistence/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+  id "application"
+}
+
+dependencies {
+	compile "org.eclipse.jgit:org.eclipse.jgit:4.7.0.201704051617-r"
+	testCompile "org.eclipse.jgit:org.eclipse.jgit.junit:4.7.0.201704051617-r"
+}
+
+mainClassName = 'org.testeditor.web.backend.persistence.ServerLauncher'
```

Comment:
We should remove the `ServerLauncher` and add the main method to the application.